javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,47 @@
+// Peter's house measuremensts
+
+let houseWidth = 8;
+let houseDepth = 10;
+let houseHeight = 10;
+let gardenSizeInM2 = 100;
```

Comment:
These can be `const` because you don't change the value after. Or you use the same variables for Julai with `let`